.Rbuildignore

```diff
@@ -27,3 +25,5 @@
 ^_pkgdown\.yml$
 ^docs$
 ^pkgdown$
+^vignettes/[^-]+.gif$
+^man/figures$
```

Comment:
This is surprising — and I'm pretty sure it will break the rendering of the README on CRAN.